HolidayList/HolidayAPIKey.txt

```diff
@@ -0,0 +1 @@
+﻿3b2cc4a9fcb58a121947815135fca92694b0f63e
```

Comment:
New file in root folder that holds the APIKey for the JSON feed.  This file needs to be added to the git ignore file, but I wanted to leave it now so it will compile when you sync to your system.